azure-pipelines.yml

```diff
@@ -0,0 +1,49 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger:
+  # batch runs this pipeline one after the other (useful when there are many contributors)
+  batch: true
```

Comment:
I'm not sure about the batch functionality nor used it before. If this some kind of extra-mile please let's push it to our tech debt backlog and discuss if it's needed

Reply:
can you please point me to the right place where "tech debt backlog" is?